conf/machine/orange-pi-zero2.conf

```diff
@@ -0,0 +1,19 @@
+#@TYPE: Machine
+#@NAME: orange-pi-zero-2
+#@DESCRIPTION: Machine configuration for the orange-pi-zero-2, based on Allwinner H616 CPU
+
+require conf/machine/include/sun50i.inc
+
+PREFERRED_PROVIDER_virtual/kernel = "linux-mainline"
+
+KERNEL_DEVICETREE = "allwinner/sun50i-h616-orangepi-zero2.dtb"
+UBOOT_MACHINE = "orangepi_zero2_defconfig"
+
+UBOOT_BINARY = "u-boot-nodtb.bin"
+SPL_BINARY = "u-boot-sunxi-with-spl.bin"
+
+MACHINEOVERRIDES:append = "sun50i:sun50i-h616:"
```

Comment:
This is wrong. after :append the should be 'space'. also sun50i is not necessary as it's SOC_ARCH and added by default. MACHINEOVERRIDES usually added like that:
`MACHINEOVERRIDES =. "sun50i-h616:"`

Reply:
Have a look at the recent changes I've just committed. There is one thing that we have to solve here. For the purpose of setting an appropriate PLATFORM variable for ATF we have to distinguish between sun50i and sun50i-h616. Previously as you commented above, MACHINEOVERRIDES was added in the machine conf file overriding sun50i and sun50i-h616. It worked. But in the recent changes I've created a separate include file for sun50i-h616 (and there added MACHINEOVERRIDES) because as far as I know there is more than one board using this chip, so in my opinion it would be better to keep a separate include file for this SOC.